### ISSUE TITLE
Media in an actor/item sheet

### ISSUE THREAD

**Issue by brunocalado** [DarKDinDoN/share-media#17]
Hi.

I'm trying to find how to sahre actor/item image. Is it only when in a description field?

I was thinking I could trigger share-media with the entity image somehow
![image](https://user-images.githubusercontent.com/4999783/172070257-b0b38720-c290-4c5a-bb54-6ca87ad0d3c6.png)

**Comment by DarKDinDoN** [comment]
@brunocalado

Hi, you can right-click an actor or an item in the right sidebar and click on the desire share button.
See: https://github.com/DarKDinDoN/share-media/releases/tag/0.0.7

Alternatively, if the token is on the canvas, you can right-click on it and click on the share button.
See: https://github.com/DarKDinDoN/share-media/releases/tag/0.0.20

Feel free to close this issue if these features are enough for you.

**Comment by brunocalado** [comment]
> @brunocalado
> 
> Hi, you can right-click an actor or an item in the right sidebar and click on the desire share button. See: https://github.com/DarKDinDoN/share-media/releases/tag/0.0.7
> 
> Alternatively, if the token is on the canvas, you can right-click on it and click on the share button. See: https://github.com/DarKDinDoN/share-media/releases/tag/0.0.20
> 
> Feel free to close this issue if these features are enough for you.

The sidebar worked.

But, I can't find the HUD
![image](https://user-images.githubusercontent.com/4999783/172071727-11e810d0-731d-4a9b-b492-acf01dbf6230.png)

**Comment by brunocalado** [comment]
Find the Culprit
![image](https://user-images.githubusercontent.com/4999783/172071779-b6597676-71f6-446b-b8c0-49ea77cf6353.png)

**Comment by DarKDinDoN** [comment]
@brunocalado do you have an error printed in the console?

**Comment by brunocalado** [comment]
> @brunocalado do you have an error printed in the console?

Your module is not generanting any error.

**Comment by brunocalado** [comment]
Wait. When I right click there is a error.
![image](https://user-images.githubusercontent.com/4999783/172072584-67be922a-4877-458c-9fc9-9853c4cc943c.png)

**Comment by brunocalado** [comment]
```
Error: This is not a registered game setting
    at ClientSettings.get (foundry.js:2150:42)
    at main.js:26:43
    at Hooks._call (foundry.js:294:14)
    at Hooks.call (foundry.js:279:33)
    at TokenHUD._render (foundry.js:3117:13)
    at async TokenHUD._render (foundry.js:52680:4)
```

**Comment by DarKDinDoN** [comment]
@brunocalado yeah, my mistake.
Deleted this setting in the previous release.
New release should fix this: [0.0.24](https://github.com/DarKDinDoN/share-media/releases/tag/0.0.24)

Is ok for you?

**Comment by brunocalado** [comment]
> @brunocalado yeah, my mistake. Deleted this setting in the previous release. New release should fix this: [0.0.24](https://github.com/DarKDinDoN/share-media/releases/tag/0.0.24)
> 
> Is it ok for you?

Yes. Thank you.
